todolistproject/src/main/java/com/todo/todolist/SeedData.java

```diff
@@ -0,0 +1,65 @@
+package com.todo.todolist;
+
+import com.todo.todolist.model.Role;
+import com.todo.todolist.model.Todo;
+import com.todo.todolist.model.User;
+import com.todo.todolist.model.UserRoles;
+import com.todo.todolist.repo.RoleRepository;
+import com.todo.todolist.repo.TodoRepository;
+import com.todo.todolist.repo.UserRepository;
+import org.springframework.boot.CommandLineRunner;
+import org.springframework.stereotype.Component;
+import org.springframework.transaction.annotation.Transactional;
+
+import java.util.ArrayList;
+
+@Transactional
+@Component
+public class SeedData implements CommandLineRunner
+{
+    private RoleRepository rolerepos;
+    private UserRepository userrepos;
+    private TodoRepository todorepos;
+
+    public SeedData(RoleRepository rolerepos, UserRepository userrepos, TodoRepository todorepos)
+    {
+        this.rolerepos = rolerepos;
+        this.userrepos = userrepos;
+        this.todorepos = todorepos;
+    }
+
+    @Override
+    public void run(String[] args) throws Exception
+    {
+        Role r1 = new Role("admin");
+        Role r2 = new Role("user");
+
+        ArrayList<UserRoles> admins = new ArrayList<>();
+        admins.add(new UserRoles(new User(), r1));
+        admins.add(new UserRoles(new User(), r2));
+
+        ArrayList<UserRoles> users = new ArrayList<>();
+        users.add(new UserRoles(new User(), r2));
+
+        rolerepos.save(r1);
+        rolerepos.save(r2);
+
+        User u1 = new User("barnbarn", "ILuvM4th!", users);
```

Comment:
There seems to be a problem with everything from this point on. Be sure to check your user and todo constructors